docs/src/pages/components/badge.mdx

```diff
@@ -1,2 +1,168 @@
+import { Link } from 'gatsby';
+
 # Badge
 
+The Badge component displays a dot or a circle with a count. It is normally used to indicate pending items (items in cart, unread messages, etc.)
+
+## Usage
+
+There are three sizes: `dot` (default) and `small` are meant to be used with an icon (Anchor comes with <Link to="/components/icon">its own set of icons</Link>). `large` is meant to be used standalone.
+
+### Size dot with an icon
+```tsx
+<div style={{ padding: '2rem 1rem' }}>
+    <Badge
+        borderColor={'white'}
+        borderColorHover={'white'}
+        count={1}
+        offsetBottom={1.2}
+        offsetLeft={0.9}
+    >
+        <Cart scale="lg" />
+    </Badge>
+</div>
+```
+
+### Size small with an icon
+
+To see how `showZero` works, try setting `count` to `{0}`. The badge should disappear. If you set `showZero` to `{true}`, however, the badge should display a `0`.
+
+```css
```

Comment:
I'm currently working on a way to have code blocks that work w/o triggering the live code editor. For now though, we can't have code blocks that aren't interactive. Plus they are always interpreted as a component, so this css is producing an error. You'll need to either somehow integrate this bit into the component you have in the following example or figure out another way to explain it.

Reply:
I used your new example code block ✨ 

It was helpful for explaining the `isParentHovered` prop 👍